terasoluna-gfw-common-libraries/terasoluna-gfw-validator/src/test/java/org/terasoluna/gfw/common/validator/constraints/CompareTest.java

```diff
@@ -523,72 +501,67 @@ public void testSpecifyNotComparableLeft() throws Throwable {
-                        MESSAGE_VALIDATION_ERROR, "left", "right"))))));
-    }
+        {
+            form.setListProperty(Arrays.asList(new Pair(101, 100),
+                    new Pair(100, 100)));
 
-    /**
-     * all values in the collection are invalid.
-     */
-    @Test
-    public void testCollectionAllInvalid() {
-        form.setListProperty(Arrays.asList(new Pair(101, 100),
-                new Pair(100, 101)));
+            violations = validator.validate(form);
+            assertThat(violations, containsInAnyOrder( //
+                    allOf( //
+                            hasProperty("propertyPath", hasToString(
+                                    "listProperty[0].<list element>.left")), //
+                            hasProperty("message", is(String.format(
+                                    MESSAGE_VALIDATION_ERROR, "left",
+                                    "right"))))));
+        }
 
-        violations = validator.validate(form);
-        assertThat(violations.size(), is(2));
-        assertThat(violations, containsInAnyOrder( //
-                allOf( //
-                        hasProperty("propertyPath", hasToString(
-                                "listProperty[0].<list element>.left")), //
-                        hasProperty("message", is(String.format(
-                                MESSAGE_VALIDATION_ERROR, "left", "right")))), //
-                allOf( //
-                        hasProperty("propertyPath", hasToString(
-                                "listProperty[1].<list element>.left")), //
-                        hasProperty("message", is(String.format(
-                                MESSAGE_VALIDATION_ERROR, "left", "right"))))));
+        {
+            form.setListProperty(Arrays.asList(new Pair(100, 100),
+                    new Pair(100, 101)));
+
+            violations = validator.validate(form);
+            assertThat(violations, containsInAnyOrder( //
+                    allOf( //
+                            hasProperty("propertyPath", hasToString(
+                                    "listProperty[1].<list element>.left")), //
+                            hasProperty("message", is(String.format(
+                                    MESSAGE_VALIDATION_ERROR, "left",
+                                    "right"))))));
+        }
+
+        {
+            form.setListProperty(Arrays.asList(new Pair(101, 100),
+                    new Pair(100, 101)));
+
+            violations = validator.validate(form);
+            assertThat(violations, containsInAnyOrder( //
+                    allOf( //
+                            hasProperty("propertyPath", hasToString(
+                                    "listProperty[0].<list element>.left")), //
+                            hasProperty("message", is(String.format(
+                                    MESSAGE_VALIDATION_ERROR, "left",
+                                    "right")))), allOf( //
```

Comment:
The format is broken.
It is recommended to put `//` between `),` and `allOf(` to ensure formatting.

Reply:
Thx! I fixed all with grep `allOf(`. 3feceda